docs/source/getting_started/install.rst

```diff
@@ -7,22 +7,41 @@ Installation
 Install using ``pip``
 ---------------------
 
-`pip <https://pypi.org/project/pip/>`_ is the package installer for Python.
+The standard package installer for Python is `pip <https://pypi.org/project/pip/>`_.
+
 To use PyDPF-Post with Ansys 2021 R1 or later, install the latest version
-with:
+with this command:
 
 .. code::
 
    pip install ansys-dpf-post
 
+PyDPF-Post plotting capabilities require to have `PyVista <https://pyvista.org/>`_ installed.
+To install PyDPF-Post with its optional plotting functionalities, use:
+
+.. code::
+   pip install ansys-dpf-post[plotting]
+
+For more information about PyDPF-Post plotting capabilities, see :ref:`_user_guide_plotting`.
 
-Install using a wheel file
---------------------------
 
-If you are unable to install PyDPF-Post on the host machine due to
-network isolation, download the latest wheel file from `PyDPF-Post
-GitHub <https://github.com/pyansys/pydpf-post>`_ or
-`PyDPF-Post PyPi <https://pypi.org/project/ansys-dpf-post/>`_.
+Install without internet
+------------------------
+
+If you are unable to install PyDPF-Post on the host machine using ``pip`` due to
+network isolation, download the wheelhouse corresponding to your platform and Python interpreter version
+for the latest release of PyDPF-Post from the assets section of the `latest PyDPF-Post release on GitHub <https://github.com/pyansys/pydpf-post/releases/latest>`_. 
+
+The wheelhouse is a ZIP file containing Python wheels for all the packages PyDPF-Post requires to run.
+To install PyDPF-Post using the downloaded wheelhouse, unzip the wheelhouse to a local directory,
+then use the following command from within this local directory:
+.. code::
+
+   pip install --no-index --find-links=. ansys-dpf-post
+
+Beware that PyDPF-Post wheelhouses do not include the optional plotting dependencies.
+To allow for plotting capabilities, also download the wheel corresponding to your platform and Python interpreter version
+for `PyVista <https://pypi.org/project/pyvista/#files>`_, then place it in the same previous local directory and run the command above.
```

Comment:
```suggestion
To allow for plotting capabilities, also download the `PyVista wheel <https://pypi.org/project/pyvista/#files>`_
corresponding to your platform and Python interpreter version and unzip it to the same local
directory before running the preceding command again.
```
@germa89 Is the "corresponding to your platform and Python interpreter version" necessary here? When I go to the linked PyVisata page, I see only one wheel file.

Reply:
@PipKat You are right, this may not be necessary, I will remove it.
```suggestion
To allow for plotting capabilities, also download the `PyVista wheel <https://pypi.org/project/pyvista/#files>`_
and unzip it to the same local directory before running the preceding command again.
```